### PR TITLE
Specify black line length is 88 characters

### PR DESCRIPTION
**Pull request by StellarStorm**
This is the default black setting, but some users (like me) will globally override to a different length. Better to explicitly specify.

### CODE REVIEW

**Comment by StellarStorm** [comment]
@loli this makes sense to me, but a few files got commas removed. It would be nice to have another set of eyes to make sure this makes sense before merging.

**Review by loli — APPROVED** [review]
Makes sense, thanks for catching this!

**Pull request merged**
